tests/fixtures/contains-cycle/is-cycle

```diff
@@ -0,0 +1 @@
+/Users/thoover/code/hash-for-dep/tests/fixtures/contains-cycle/
```

Comment:
I've been trying to figure out if this can be set once and recognized by git by doing 
ln -s <whatever> '..'
When I did my own version of this (because the stored one mentioned 'stefanpenner'), I used .. instead of '..' and ended up storing my own link (with 'davecombs' in the path).

Reply:
I think the easiest is to do the symlinks in code